src/components/Menu/Menu.tsx

```diff
@@ -2,23 +2,21 @@ import "./Menu.scss"
 
 import List from "../List/List";
 import { useCallback, useState } from "react";
+import { SortControl, SortOption } from "../SortControl/SortControl";
 
 const Menu = () => {
-    const [itemNames, setItemNames] = useState<string[]>(["Action", "Adventure", "Comedy", "Drama", "Sci-Fi", "Thriller", "Western"]);
+    const [itemNames, setItemNames] = useState<string[]>(["Action", "Comedy", "Drama", "Sci-Fi", "Thriller", "Western"]);
     const [preselectedItemName, setSelectedItemName] = useState<string>("Sci-Fi");
-    const onSelect = useCallback((query: string) => { console.log(query); }, [])
+    const onSelect = useCallback((query: string) => { console.log(query); }, []);
+    const onSortOptionChange = useCallback((currentOption:SortOption) => console.log(currentOption), []);
```

Comment:
looks  like this function doesn't work

Reply:
It is work. To trigger this function, you need to click on any genre in menu. Then you can see the genre name in console 
![image](https://user-images.githubusercontent.com/87666101/227279072-d0358321-deac-488e-b092-c70865cb3fcd.png)